### ISSUE TITLE
Port notebooks under model/docs to quarto

### ISSUE THREAD

**Issue by gvegayoncdc** [CDCgov/multisignal-epi-inference#34]
## Goal

Jupyter notebooks are great, but not so for Git. We want to replace them  (for the moment) with [Quarto documents](https://quarto.org).

## Context

See discussion on alternatives to Jupyter: #22 

## Required features

Currently, we have two Jupyter notebooks under `model/docs/`: getting-started and pyrenew_demo. Both need to be ported to quarto.

## Specifications

- [x] Port `getting-started`.
- [x] Port `pyrenew_demo`.
- [x] `getting-started` has a `gfm` output saved under `docs`.
- [x] `pyrenew_demo` has a `gfm` output saved under `docs`.
- [x] Check whether there's a pre-commit for un-updated md documents. If it exists, add it.

## Out of scope

- Create the rst files for the website. That will be done after #1 is completed.

**Comment by gvegayoncdc** [comment]
While looking for pre-commit hooks for quarto, I found [quartodoc](https://machow.github.io/quartodoc/get-started/overview.html), an alternative for sphinx that is based on quarto 🤯! cc @AFg6K7h4fhy2 @SamuelBrand1 @natemcintosh
